x-pack/plugins/security_solution/public/management/pages/policy/models/advanced_policy_schema.ts

```diff
@@ -72,6 +72,17 @@ export const AdvancedPolicySchema: AdvancedPolicySchemaType[] = [
       }
     ),
   },
+  {
+    key: 'linux.advanced.artifacts.global.channel',
+    first_supported_version: '8.18',
+    documentation: i18n.translate(
+      'xpack.securitySolution.endpoint.policy.advanced.linux.advanced.artifacts.global.channel',
+      {
+        defaultMessage:
+          'The release channel to use for receiving global artifacts. By default, staged roll-out is applied. Set to rapid to receive candidate artifacts as soon as available. Set to stable to only receive stable artifacts. Default: default',
```

Comment:
This confused me a bit in that I thought the default value was "staged".
Maybe add quotes around the values that can be used so that they standout from the instructions. Example: 

`The "default" is to staged roll-out. Set to "rapid" to receive candidate artifacts as soon as available. Set to "stable" to only receive stable artifacts. Default: default`

_(note: I did not suggest adding quotes to the last part of the message becuase it was placed in front of the label "Default: ... " )_

Reply:
same comment for instances below